addons/utm/static/src/js/utm_campaign_kanban_examples.js

```diff
@@ -0,0 +1,34 @@
+odoo.define('utm.campaing_kanban_examples', function (require) {
+    'use strict';
+
+    var core = require('web.core');
+    var kanbanExamplesRegistry = require('web.kanban_examples_registry');
+
+    var _lt = core._lt;
+
+    kanbanExamplesRegistry.add('utm_campaign', {
+        ghostColumns: [_lt('New'), _lt('Schedule'), _lt('Design'), _lt('Sent')],
+        apply_example_text: _lt("Use This For My Campaigns"),
+        examples: [{
+            name: _lt('Event-driven Flow'),
+            columns: [_lt('Later'), _lt('This Month'), _lt('This Week'), _lt('Running'), _lt('Sent')],
+            description: "Track incoming events (e.g. : Christmas, Black Friday, ...) and publish timely content.",
```

Comment:
You should use _lt() for the descriptions as well to make sure they are correctly translated into the user locale.